bin/strata-cli/src/recovery.rs

```diff
@@ -13,7 +13,7 @@ use bdk_wallet::{
     miniscript::{descriptor::DescriptorKeyParseError, Descriptor},
     template::DescriptorTemplateOut,
 };
-use rand::{thread_rng, Rng};
+use rand::{rngs::OsRng, RngCore};
```

Comment:
Why do we need `RngCore` here?

Reply:
It's required to provide `fill_bytes`. I prefer this paradigm for setting random byte arrays over relying on specific `gen`-based distributions.